web/geo/generate-topojson.js

```diff
@@ -1,10 +1,10 @@
 const { topology } = require('topojson-server');
-const { getJSON, writeJSON, round } = require('./utilities');
+const { getJSON, writeJSON, round, fileExists } = require('./utilities');
 const turf = require('@turf/turf');
 
 function getCenter(geojson, zoneName) {
   const geojsonFeatures = geojson.features.filter((f) => f.properties.zoneName === zoneName);
-  if (geojsonFeatures.length !== 1) {
+  if (geojsonFeatures.length !== 1 && geojsonFeatures.length !== 2) {
```

Comment:
Can you elaborate on this change? :)

Reply:
Originally there was duplicated features for an aggregated and unaggregated map. This design has been updated to only have new features for the aggregated zones so this check has been removed.